homeassistant/components/deconz/sensor.py

```diff
@@ -90,35 +90,32 @@
 class DeconzSensorDescriptionMixin(Generic[T]):
     """Required values when describing secondary sensor attributes."""
 
-    isinstance_fn: Callable[[T], bool]
     update_key: str
     value_fn: Callable[[T], datetime | StateType]
 
 
 @dataclass
-class DeconzSensorDescription(
-    SensorEntityDescription, DeconzSensorDescriptionMixin[T], Generic[T]
-):
+class DeconzSensorDescription(SensorEntityDescription, DeconzSensorDescriptionMixin[T]):
     """Class describing deCONZ binary sensor entities."""
 
-    common: bool = False
+    instance_check: type | None = None
```

Comment:
You can even improve here:
```suggestion
    instance_check: type[T] | None = None
```

Reply:
Thanks, resolved!